scripts/module/stadium.nut

```diff
@@ -0,0 +1,40 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+
+function SetStadiumBillboard(offset) {
+  if (billboard) {
+    billboard.Delete();
+  }  
+  billboard <- CreateObject(BillboardsBegin + offset, 1, -1037.08, 1340.258, 36.552, 255);
+  MessagePlayer("Changed to stadium billboard offset " + offset, player);
+}
+
+function onScriptLoad() {
+  SetStadiumBillboard(0);
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  if (! player.Admin ) {
+    return;
+  }
+  cmd = cmd.tolower();
+  if (cmd == "ssb" || cmd == "setstadiumbillboard") {
+    if ( text ) {
+      local arg = 5;
+      if (IsNum(text)){
+        arg = text.tointeger();
+      }
+      if (arg < 3 && arg >= 0) {
+        SetStadiumBillboard(arg);
+      } else if (arg == -1) {
+        billboard.Delete();
```

Comment:
I'd handle this as part of the `SetStadiumBillboard` API, because if we export it, the `billboard` variable will no longer be accessible. It will be an internal implementation detail.

As such, I'd probably also check what special value vc-mp uses for disabling things (maybe `false`, maybe `null`, maybe `0` or maybe `-1`).